apps/wallet/src/ui/app/shared/field-label/index.tsx

```diff
@@ -13,7 +15,12 @@ export type FieldLabelProps = {
 export default function FieldLabel({ txt, children }: FieldLabelProps) {
     return (
         <label className={st.container}>
```

Comment:
Converting this to tailwind will allow to remove `FieldLabel.module.scss` as well and not have there unused classes

Reply:
done